source/core/slang-downstream-compiler.h

```diff
@@ -109,12 +128,15 @@ class DownstreamCompiler: public RefObject
         Clang,
         SNC,
         GHS,
+        NVRTC,
         CountOf,
     };
     enum class SourceType
     {
         C,              ///< C source
         CPP,            ///< C++ source
+        CUDA,           ///< The CUDA language
```

Comment:
I suppose that very technically the language is "CUDA C++" and not just "CUDA," but I don't think we'll have any problems down the line from just using the shorter name (we aren't going to add a CUDA Fortran back-end any time soon).